src/docs/openshift-projects-and-access/install-the-oc-command-line-tool.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Install the oc command line tool
+
+slug: install-the-oc-command-line-tool
+
+description: Describes how to install the oc command line tool for mac and windows 
+
+keywords: OpenShift, command line, oc, brew, cli
+
+page_purpose: Describes the installation of the oc command line tool for users preparing to use OpenShift 
+
+audience: technical lead, openshift 101 students, developers
+
+author: Matt Spencer  
+
+editor: Pilar Solares
+
+content_owner: Faisal Hamood
+
+sort_order: 4
+---
+
+# Install the oc command line tool
+Last updated: **May 30, 2023**
+
+Users can interact with OpenShift via the `oc` command line tool. This can be installed on your local machine, or accessed via the web console. It is important to keep your version of `oc` up to date to match the version of `oc` on the cluster. 
+
+## On this page
+* <a href="section-1">**Install oc on a mac**</a>  
+* <a href="section-2">**Install oc on windows**</a>   
+* <a href="section-3">**Use web terminal**</a>  
+* <a href="section-4">**Related pages**</a>
+
+<!-- ### End of On this page -->
+
+ <h2 id="section-1">Install oc on a mac</h2> 
```

Comment:
Is there a reason for using html instead of just markdown?
```suggestion
## Install oc on a mac
```
You can link to sections in markdown like this
```
* [**Install oc on a mac**](#install-oc-on-a-mac)
```

Reply:
@Pilargit12 please note the format that Steven suggests here for how you can link to a heading markdown in markdown.